Language-Version-History.md

```diff
@@ -3,7 +3,8 @@ Features Added in C# Language Versions
 
 # C# 10.0 - .NET 6 and Visual Studio 2022 version 17.0
 
-- [Record structs](https://github.com/dotnet/csharplang/blob/master/proposals/csharp-10.0/record-structs.md) and `with` expressions on structs (`record struct Point(int X, int Y);`, `var newPoint = point with { X = 100 };`).
+- [Record structs](https://github.com/dotnet/csharplang/blob/master/proposals/csharp-10.0/record-structs.md) (`record struct Point(int X, int Y);`, `var newPoint = point with { X = 100 };`).
+- [With expression](https://docs.microsoft.com/dotnet/csharp/language-reference/operators/with-expression) on structs and anonymous types.
```

Comment:
I don't think we linked to docs.microsoft.com from this document before. I would have linked to https://github.com/dotnet/csharplang/blob/main/proposals/csharp-10.0/record-structs.md#allow-with-expression-on-structs

Reply:
we do, almost in all references for C# 6 and previous versions we link to docs.microsoft.com.
but, I changed the reference since features > C# 6 are using github link.